src/library/Cardano/CoinSelection/RandomImprove.hs

```diff
@@ -395,5 +397,10 @@ mkChange (Output _ (Coin out)) inps =
 -- Utilities
 --------------------------------------------------------------------------------
 
+-- | Compute distance between two numeric values |a - b|
+distance :: (Ord a, Num a) => a -> a -> a
```

Comment:
this could also to the `Internal` or `cardano-aux` ;-)

Reply:
Ah, but that's not exported by `RandomImprove`, so it's fine to leave that where it is.